app/src/test/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinScreenLogicTest.kt

```diff
@@ -420,14 +420,16 @@ class ForgotPinConfirmPinScreenLogicTest {
   }
 
   private fun setupController(pin: String) {
+    val viewEffectHandler = ForgotPinConfirmPinViewEffectHandler(uiActions)
     val effectHandler = ForgotPinConfirmPinEffectHandler(
         userSession = userSession,
         currentUser = Lazy { loggedInUser },
         currentFacility = Lazy { facility },
         resetUserPin = resetUserPin,
         syncAndClearPatientData = syncAndClearPatientData,
         schedulersProvider = TestSchedulersProvider.trampoline(),
-        uiActions = uiActions
```

Comment:
That can be done as a different PR. We ideally don't want to do this here where we are also migrating it to view effects.

Reply:
But don't you think while we are doing such migrations covering the codebase with such small tests can lead to more test coverage as we are not planning to get back at this anyways?

Reply:
Not really, these are regression tests from our older architecture. Unless we are changing the areas of specific things where we require the tests to be modified as well. We should not migrate the tests like this. 

One scenario where we do migrate tets is when most of the cases now have effect handler tests. Then we can move the remaining ones and remove the older tets. 

In the current PR we are updating view effects, so we can add effect handler tests. But we cannot remove the logic tests yet. Because we need update and init tests as well.

Reply:
Also, our test coverage for this is already there. Because we are using `MobiusTestFixture` which lets us tests run on our Mobius implementation.

Reply:
umm got it 👍🏻